src/util/circular_buffer.rs

```diff
@@ -0,0 +1,29 @@
+//! CircularBuffer
+use std::collections::VecDeque;
+///A CircularBuffer that drops the oldest element inserted when full.
```

Comment:
Please add one newline after the mod comment and one after the import.

Reply:
Fixed